src/datumaro/plugins/data_formats/roboflow/importer.py

```diff
@@ -214,13 +214,15 @@ class RoboflowMulticlassImporter(RoboflowCocoImporter):
 
 
 @extra_deps("tensorflow")
-class RoboflowTfrecord(Importer):
+class RoboflowTfrecordImporter(Importer):
     FORMAT = "roboflow_tfrecord"
 
     @classmethod
     def find_sources(cls, path):
         sources = cls._find_sources_recursive(
-            path=path, ext=".tfrecord", extractor_name="roboflow_tfrecord", filename="cells"
+            path=path,
+            ext=".tfrecord",
+            extractor_name="",
```

Comment:
Is it ok to set `extractor_name=""`?

Reply:
Yes it was, but it would be better to specify the extractor_name again. I will fix this.

Reply:
fixed at 95fad54349fc7fbe06a3474e2ef43f9e6b0384fe